cli/args/flags.rs

```diff
@@ -1122,6 +1122,8 @@ impl Flags {
 
 static ENV_VARIABLES_HELP: &str = cstr!(
   r#"<y>Environment variables:</>
+<y>Docs:</> <c>https://docs.deno.com/go/env-vars</>
```

Comment:
When I request this URL, I receive a `404` status response. Is the intention for it to resolve with actual content? Is `https://docs.deno.com/go/*` some sort of redirect endpoint for [URL shortening](https://en.wikipedia.org/wiki/URL_shortening) purposes?

Reply:
there is a PR to add it in docs.deno.com already